src/intTest/resources/failure_uat_data/NHS004/screening_interchange_parsing_failure_with_nhsack.edifact.dat

```diff
@@ -0,0 +1,43 @@
+UNA:+.? '
+UNB+UNOC:3+000000004400001+000000024600002+091113:1519+34++SCRN_BCS_MEDRPT_V4++1'
```

Comment:
This doesn't look right `SCRN_BCS_MEDRPT_V4`
Check other files as well

Reply:
Matthew brought this up as well, it should be MEDRPT but changing it requires some changes to other integration tests and the NhsAckProducerService in main, so I have a separate pull request ready to go to fix that after this is merged.